sdks/python/flojoy-cloud/flojoy_cloud/test_sequencer.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import os
+import pandas as pd
+import tempfile
+from .measurement import MeasurementData
+
+
+# ------ Public ------
+
+__all__ = [
+    "export",
+    "_get_most_recent_data",
+    "_set_output_loc",
+]
+
+
+def export(data: MeasurementData):
+    """Export data so it can be retrieved by the test sequencer"""
+    output_dir, prefix_file = __get_location()
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
+    elif isinstance(data, pd.DataFrame):
+        data.to_csv(os.path.join(output_dir, prefix_file + DATAFRAME))
+    elif isinstance(data, bool):
+        with open(os.path.join(output_dir, prefix_file + BOOLEAN), "w") as f:
+            f.write(str(data))
+    elif isinstance(data, int):
+        with open(os.path.join(output_dir, prefix_file + INT), "w") as f:
+            f.write(str(data))
+    elif isinstance(data, float):
+        with open(os.path.join(output_dir, prefix_file + FLOAT), "w") as f:
+            f.write(str(data))
+    else:
+        raise TypeError(f"Unsupported data type: {type(data)}")
```

Comment:
This will skip writing the file on the first data export if the output dir doesn't exist, should just be an `if` on line 22

Reply:
Oh nice catch